keyboards/moonlander/moonlander.c

```diff
@@ -42,34 +42,40 @@ void moonlander_led_task(void) {
         ML_LED_1(false);
         ML_LED_2(false);
         ML_LED_3(false);
+        #ifndef HALFMOON
         ML_LED_4(false);
         ML_LED_5(false);
         ML_LED_6(false);
+        #endif
 
         ML_LED_1(true);
         wait_ms(250);
         ML_LED_2(true);
         wait_ms(250);
         ML_LED_3(true);
         wait_ms(250);
+        #ifndef HALFMOON
         ML_LED_4(true);
         wait_ms(250);
         ML_LED_5(true);
         wait_ms(250);
         ML_LED_6(true);
         wait_ms(250);
+        #endif
         ML_LED_1(false);
         wait_ms(250);
         ML_LED_2(false);
         wait_ms(250);
         ML_LED_3(false);
         wait_ms(250);
+        #ifndef HALFMOON
```

Comment:
I think this may work better in general.  This should also translate to better behavior for the regular board, if the right half is disconnected, creating a more consistent behavior.
```suggestion
        if (is_transport_connected()) {
```